guides/source/routing.md

```diff
@@ -1321,27 +1355,27 @@ namespace :admin do
 end
 ```
 
-Calling `draw(:admin)` inside the `Rails.application.routes.draw` block itself will try to load a route
-file that has the same name as the argument given (`admin.rb` in this example).
-The file needs to be located inside the `config/routes` directory or any sub-directory (i.e. `config/routes/admin.rb` or `config/routes/external/admin.rb`).
+Calling `draw(:admin)` inside the `Rails.application.routes.draw` block itself
+will try to load a route file that has the same name as the argument given
+(`admin.rb` in this example). The file needs to be located inside the
+`config/routes` directory or any sub-directory (i.e. `config/routes/admin.rb` or
+`config/routes/external/admin.rb`).
 
-You can use the normal routing DSL inside the `admin.rb` routing file, but you **shouldn't** surround it with the `Rails.application.routes.draw` block like you did in the main `config/routes.rb` file.
+NOTE: You can use the normal routing DSL inside a secondary routing file such as `admin.rb`, but *do not* surround it with the `Rails.application.routes.draw` block. That should be used in the main `config/routes.rb` file only.
 
 [`draw`]: https://api.rubyonrails.org/classes/ActionDispatch/Routing/Mapper/Resources.html#method-i-draw
 
-### Don't Use This Feature Unless You Really Need It
-
-Having multiple routing files makes discoverability and understandability harder. For most applications - even those with a few hundred routes - it's easier for developers to have a single routing file. The Rails routing DSL already offers a way to break routes in an organized manner with `namespace` and `scope`.
+NOTE: Don't use the `draw` feature unless you really need it. Having multiple routing files make it harder to discover routes in one place. For most applications - even those with a few hundred routes - it's easier for developers to have a single routing file. The Rails routing DSL already offers a way to break routes in an organized manner with `namespace` and `scope`.
 
 
-Inspecting and Testing Routes
------------------------------
+Inspecting Routes
+-----------------
 
-Rails offers facilities for inspecting and testing your routes.
+Rails offers a few different ways of inspecting and testing your routes.
 
 ### Listing Existing Routes
 
-To get a complete list of the available routes in your application, visit <http://localhost:3000/rails/info/routes> in your browser while your server is running in the **development** environment. You can also execute the `bin/rails routes` command in your terminal to produce the same output.
+To get a complete list of routes available in an application, visit `http://localhost:3000/rails/info/routes` in the **development** environment. You can also execute the `bin/rails routes` command in your terminal to get the same output.
```

Comment:
Do you think it will be useful to mention that the `_path` and the `_url` helpers can be derived for a route by adding them as suffixes to the route name? I've seen someone mention it like that in a blog post but maybe it's too obvious - I'm not sure.

Reply:
Not sure what you mean...maybe you can give an example?

Reply:
I see! Let me see if I can include that in a concise way.

Reply:
Done!